Tasks/DownloadPackageV0/download.ts

```diff
@@ -221,7 +225,14 @@ export async function unzip(zipLocation: string, unzipLocation: string): Promise
 		}
 
 		tl.debug('Extracting ' + zipLocation + ' to ' + unzipLocation);
-
+#if NODE20
```

Comment:
Why only on node 20?

Reply:
decompress-zip package is incompatible with Node17+, so we only need to replace it in Node20 version, the last node version node 16 version is good.